src/Telegram.Bot/Extensions/HttpContentExtensions.cs

```diff
@@ -21,6 +21,7 @@ internal static class HttpContentExtensions
 
         // It will be dispose of after the request is made
 #pragma warning disable CA2000
+        inputFile.Content.Position = 0;
```

Comment:
Not every stream is rewindable. We need to think of another solution

Reply:
One idea is to make a buffer stream using `MemoryStream`, though I'm not sure how to deal with disposing of that stream

Reply:
What if we cache file content in Lazy<whatever> field and reuse it on consequent calls?

Reply:
So, I was able to cache content, but it'll require quite a significant change:

```csharp
public ctor()
{
    cachedStreamContent = new(Do);
}

private readonly Lazy<Task<StreamContent?>> cachedStreamContent;

private async Task<StreamContent?> Do()
{
    HttpContent? content =  ...;
    var stream = await content!.ReadAsStreamAsync().ConfigureAwait(false);
    return new StreamContent(stream);
}

public override Task<StreamContent?> HttpContent() => cachedStreamContent.Value;
```

BTW .NET 6+ has synchronous `ReadAsStream()` method.

Reply:
#1339

Reply:
#1341